tests/unit/testsuite.js

```diff
@@ -305,11 +305,11 @@ window.domEqual = function( selector, modifier, message ) {
 			result = {};
 		$.each( properties, function( index, attr ) {
 			var value = elem.prop( attr );
-			result[ attr ] = value !== undefined ? value : "";
+			result[ attr ] = value != null ? value : "";
```

Comment:
And looking at the broader context, `domEqual` might still work if you stopped doing this empty-string normalization altogether and just assigned results from `.prop` and `.attr` directly.